services-directions/src/main/java/com/mapbox/api/directions/v5/MapboxDirections.java

```diff
@@ -339,7 +341,7 @@ public static Builder builder() {
      * @param usePost true to use POST method or false and null for GET method
      * @return this builder for chaining options together
      */
-    public abstract Builder usePostMethod(@Nullable  Boolean usePost);
+    public abstract Builder usePostMethod(@Nullable Boolean usePost);
```

Comment:
could you cut a ticket to add a java linter to mapbox-java library

Reply:
Checkstyle is configured in https://github.com/mapbox/mapbox-java/tree/main/config/checkstyle, looks like it's less strict than ktlint that we use in other projects though. Feel free to make any changes directly in there if you think that we need them.